tests/apollo/test_skvbc_long_running.py

```diff
@@ -44,9 +50,39 @@ class SkvbcLongRunningTest(unittest.TestCase):
     @with_trio
     @with_bft_network(start_replica_cmd,
                       selected_configs=lambda n, f, c: n == 7)
-    async def test_stability(self, bft_network):
+    @verify_linearizability
+    async def test_stability(self, bft_network, tracker):
         bft_network.start_all_replicas()
-        with trio.move_on_after(seconds=ONE_HOUR_IN_SECONDS):
+        with trio.move_on_after(seconds=4*ONE_HOUR_IN_SECONDS):
             while True:
-                await SkvbcTest().test_get_block_data(bft_network=bft_network, already_in_trio=True)
-                trio.sleep(seconds=10)
+                await SkvbcTest().test_get_block_data\
+                    (bft_network=bft_network, already_in_trio=True, tracker=tracker)
+                self.start_all_replicas(bft_network)
```

Comment:
I don't understand. Who stops the replicas, so that you need to start them?
Again, the purpose of the long run tests is the replicas are never stopped...